packages/node/src/integrations/console.ts

```diff
@@ -49,6 +49,7 @@ function createConsoleWrapper(level: string): (originalConsoleMethod: () => void
         sentryLevel = Severity.Log;
     }
 
+    /* eslint-disable prefer-rest-params */
```

Comment:
It's not letting me do this as a suggestion (I guess because it spans a part which it hides by default), but since it's an easy fix IMHO we should just change `arguments` to `...args` and not ignore the rule.

Reply:
Will circle back to this in a PR afterwards to not increase the scope of this PR